### PR TITLE
SPCTR-3374 Fix: Global - Resolved an issue where container background images were not rendering properly during site migration.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3374

### Screenshots

- https://d.pr/v/4YScH0

### Types of changes

- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Added to the card

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by sandeshjangam — APPROVED** [review]

**Review by kudaleganesh — APPROVED** [review]

File: classes/class-uagb-loader.php [hunk 1]
```diff
@@ -454,6 +455,41 @@ public function add_uagb_blocks_to_excerpt( $excerpt_blocks, $blocks_to_add ) {
 			// If either parameter is not an array, return the original excerpt blocks.
 			return $excerpt_blocks;
 		}
+
+		/**
+		 * Generate assets on migration.
+		 *
```
Comment (adi3890) [inline classes/class-uagb-loader.php:461]: please add since tag here.

File: classes/class-uagb-loader.php [hunk 2]
```diff
@@ -75,6 +75,7 @@ public function __construct() {
 			add_action( 'plugins_loaded', array( $this, 'load_plugin' ) );
 
 			add_action( 'init', array( $this, 'init_actions' ) );
+			add_action( 'init', array( $this, 'get_regenerate_assets_on_migration' ) );
```
Comment (adi3890) [inline classes/class-uagb-loader.php:78]: can we call this function in init_actions it self

File: classes/class-uagb-loader.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-loader.php:461]: Done

File: classes/class-uagb-loader.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-loader.php]: Done

**Review by rahulvarma722 — APPROVED** [review]

**Comment by AbhijitNage123** [comment]
@adi3890 Created a new PR with similar changes and rise to zipwp base branch. Here it is - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3497. Closing current PR now.
cc - @sandeshjangam @kudaleganesh @rahulvarma722

**Pull request closed without merging**